wagtail/contrib/search_promotions/models.py

```diff
@@ -76,6 +121,13 @@ class QueryDailyHits(models.Model):
     def garbage_collect(cls, days=None):
         """
         Deletes all QueryDailyHits records that are older than a set number of days
+
+        Args:
+           - days(int, optional): Number of days to keep.
+
+        Attributes:
+           - days(int): Number of days to keep. Default is settings.WAGTAILSEARCH_HITS_MAX_AGE
+           - min_date(datetime.date): Earliest date to keep.
```

Comment:
```suggestion
```

This code is not changed at all by this PR, no need to change any docstrings here.